### DISCUSSION TITLE
Evaluation on coco dataset

### DISCUSSION THREAD

**Discussion by ranjanABHI** [open-mmlab/mmpose#1674] [Q&A]
Guys I am trying to find keypoint mAP evaluation result on COCO datasets for various pose estimation models.
Is there any api in mmpose in which we can input coco dataset, annotation file, pose estimation model config files, detection config(in case of top down approach) as a input and we can get mAP results in return. 
I tried to find ways in mmpose documentation but not getting. 
If anyone have any idea how to do it please help.

**Comment by Ben-Louis** [comment]
Hi, thanks for using MMPose. There will be a new repository for evaluation with custom models next month. 
This new repo will support evaluation of custom models on datasets included in MMPose and other OpenMMLab toolkits. Please stay tuned :D
